src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/config.html.twig

```diff
@@ -1,28 +1,35 @@
 {% extends '@WebProfiler/Profiler/layout.html.twig' %}
 
 {% block toolbar %}
-    {# Symfony Logo #}
+
+    {# Symfony Information #}
+    {% if 'unknown' == collector.symfonyState %}
+        {% set block_status = '' %}
+        {% set symfony_version_status = 'Unable to retrieve information about the Symfony version.' %}
+    {% elseif 'eol' == collector.symfonyState %}
+        {% set block_status = 'red' %}
+        {% set symfony_version_status = 'This Symfony version will no longer receive security fixes.' %}
+    {% elseif 'eom' == collector.symfonyState %}
+        {% set block_status = 'yellow' %}
+        {% set symfony_version_status = 'This Symfony version will only receive security fixes.' %}
+    {% elseif 'dev' == collector.symfonyState %}
+        {% set block_status = 'yellow' %}
+        {% set symfony_version_status = 'This Symfony version is still in the development phase.' %}
+    <span class="sf-toolbar-status sf-toolbar-status-yellow" title="">
```

Comment:
this should indeed be removed. It's now also causing the close button to not work properly anymore.